turbinia/workers/__init__.py

```diff
@@ -99,6 +148,16 @@ def add_evidence(self, evidence):
     """
     self.evidence.append(evidence)
 
+  def save_local_file(self, file_):
+    """Saves local file by writing to all non-local output writers.
```

Comment:
Yeah, maybe I can come up with a name that's clearer.  It will take a file that already exists locally, and then writes it to all output writers except for the local one (since it already exists locally).  Let me know if you have suggestions for a better name.